docs/cvdr.md

```diff
@@ -29,29 +27,69 @@ sudo apt install cuttlefish-cvdremote
 cvdr --help
 ```
 
+### Manual build
+
+To build `cvdr` manually, please run:
+```bash
+git clone https://github.com/google/cloud-android-orchestration.git
+cd cloud-android-orchestration # Root directory of git repository
+go build ./cmd/cvdr
+```
+
 ## Configure cvdr
 
-Please check and modify the configuration file(`~/.config/cvdr/cvdr.toml`).
+Please check and modify the configuration file (defaults to `~/.config/cvdr/cvdr.toml`).
 See either 
 [build/debian/cuttlefish_cvdremote/host/etc/cvdr.toml](/build/debian/cuttlefish_cvdremote/host/etc/cvdr.toml)
 or
 [scripts/on-premises/single-server/cvdr.toml](/scripts/on-premises/single-server/cvdr.toml)
 as examples of how to write a configuration file.
 
-## Use cvdr
 
-Let's assume using the latest Cuttlefish x86_64 image enrolled in
-[ci.android.com](https://ci.android.com/).
+## Run CVD
+
+
+```bash
+CVDR_USER_CONFIG_PATH=/path/to/cvdr.toml ./cvdr --help
```

Comment:
We should also make `cvdr` try to read config files from the default locations when the variables are not defined.